test/e2e/helpers.go

```diff
@@ -60,8 +59,8 @@ type deploymentsClientAdapter struct {
 }
 
 // Get fetches the deployment named by the key and updates the provided object.
-func (c deploymentsClientAdapter) Get(ctx context.Context, key client.ObjectKey, obj runtime.Object) error {
-	deployment, err := c.client.Get(key.Name, metav1.GetOptions{})
+func (c deploymentsClientAdapter) Get(ctx context.Context, key client.ObjectKey, obj client.Object) error {
+	deployment, err := c.client.Get(context.Background(), key.Name, metav1.GetOptions{})
```

Comment:
use ctx here and in the rest of this file

Reply:
yup